assets/javascript/dropdown.js

```diff
@@ -0,0 +1,20 @@
+/* When the user clicks on the button, 
+    toggle between hiding and showing the dropdown content */
+    function myFunction() {
+        document.getElementById("myDropdown").classList.toggle("show");
+    }
+    
+    // Close the dropdown menu if the user clicks outside of it
+    window.onclick = function(event) {
+      if (!event.target.matches('.dropbtn')) {
+    
+        var dropdowns = document.getElementsByClassName("dropdown-content");
+        var i;
+        for (i = 0; i < dropdowns.length; i++) {
```

Comment:
inline var within the for loop 😊 

should be for (var i = 0; i < dropdowns.length; i++) { ... }